specification/msi/resource-manager/Microsoft.ManagedIdentity/preview/2021-09-30-preview/ManagedIdentity.json

```diff
@@ -0,0 +1,816 @@
+        "x-ms-examples": {
+          "IdentityListByResourceGroup": {
+            "$ref": "./examples/IdentityListByResourceGroup.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The list of userAssignedIdentities was retrieved and returned successfully.",
+            "schema": {
+              "$ref": "#/definitions/UserAssignedIdentitiesListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{resourceName}/listAssociatedResources": {
+      "post": {
+        "operationId": "UserAssignedIdentities_ListAssociatedResources",
+        "description": "Lists the associated resources for this identity.",
+        "x-ms-examples": {
+          "IdentityListAssociatedResources": {
+            "$ref": "./examples/IdentityListAssociatedResources.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/FilterParameter"
+          },
+          {
+            "$ref": "#/parameters/OrderbyParameter"
+          },
+          {
+            "$ref": "#/parameters/TopQueryParameter"
+          },
+          {
+            "$ref": "#/parameters/SkipQueryParameter"
+          },
+          {
+            "$ref": "#/parameters/SkipTokenParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. List of resources associated with this identity.",
+            "schema": {
+              "$ref": "#/definitions/AssociatedResourcesListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
```

Comment:
>nextLink [](http://example.com/codeflow?start=27&length=8)

Paging isn't defined for POST APIs. Not sure whether the SDK clients will GET the nextLink or POST the nextLink. You should ensure sure your RP does the right thing for all the clients.

Reply:
we have an issue https://github.com/Azure/autorest/issues/4454 to track this.

Reply:
The service will only accept POST requests, because this is modelled as an ARM action as described in the private PR. If the client library generates a GET request for nextLink this will be denied.

Reply:
verified that the following use POST:

 java (see https://github.com/AzureSDKAutomation/azure-sdk-for-java/pull/13168/files#diff-ba6556ebfb626e31e46bf32a62b2fb742ca95bbfc4df98fe78ce8837aa13651eR97)
 go (see https://github.com/AzureSDKAutomation/azure-sdk-for-go/pull/5242/files#diff-fce9b8832f0a7045630abc0cea4c9d73a7782c298920f789dd54c204a21d033aR246)
 net (see https://github.com/AzureSDKAutomation/azure-sdk-for-net/pull/7728/files#diff-1c0ac9cfa3da67bdd026b6103cd8ce2fe89326623b6006b73fe15f4297b8d24fR549)
 python (see https://github.com/AzureSDKAutomation/azure-sdk-for-python/pull/9425/files#diff-66e6766f47281cc69bc280eea16813c47ea3323b61762d54b9edf6cc6cc9f7f2R137)
 js (see https://github.com/AzureSDKAutomation/azure-sdk-for-js/pull/7966/files#diff-1f5caf1baebd30e4926a80f8447f674f99ffdf9d120f3e2d3cffa97211c7015bR432)